src/core_plugins/kibana/public/management/app.html

```diff
@@ -18,10 +18,10 @@
         </span>
         <a
           ng-if="sectionName"
-          ng-repeat="item in section.items.inOrder"
+          ng-repeat="item in section.visibleItems"
           class="kuiLocalTab"
-          ng-class="{ 'kuiLocalTab-isSelected': item.active, 'kuiLocalTab-disabled': !item.url }"
-          kbn-href="{{::item.url}}"
+          ng-class="{ 'kuiLocalTab-isSelected': item.active, 'kuiLocalTab-disabled': item.disabled || !item.url }"
+          kbn-href="{{::item.disabled ? '' : item.url}}"
```

Comment:
This doesn't actually disable the link, and clicking on it will cause a page refresh. Wouldn't it be better to add a condition to `ng-if` so that it doesn't show up at all? Or maybe replace it with a `<span>` when it's disabled?